tm_examples/bert_score-own_model.py

```diff
@@ -0,0 +1,98 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""An example of how to use BERTScore with a user's defined/own model and tokenizer.
+
+To run: python bert_score-own_model.py
+"""
+
+from pprint import pprint
+from typing import Dict, List, Union
+
+import torch
+import torch.nn as nn
+
+from torchmetrics import BERTScore
+
+_NUM_LAYERS = 2
+_MODEL_DIM = 4
+_NHEAD = 2
+_MAX_LEN = 6
+
+
+class UserTokenizer:
+    def __init__(self):
+        self.cls_token = "<cls>"
+        self.sep_token = "<sep>"
+        self.pad_token = "<pad>"
```

Comment:
```suggestion
    CLS_TOKEN = "<cls>"
    SEP_TOKEN = "<sep>"
    PAD_TOKEN = "<pad>"

    def __init__(self):
```
are they class constants or spec for each instance?